src/ansys/fluent/core/services/health_check.py

```diff
@@ -53,10 +53,8 @@ class HealthCheckService:
     class Status(Enum):
```

Comment:
Shall we move "Status" outside of "HealthCheckService"? Actually, I'm a bit on the edge about this, want other's views.

Reply:
We can move this to `pyfluent_enums.py`.

Reply:
@prmukherj @hpohekar The naming of that enums module must be misleading. Aren't those meant to for launch arguments? We wouldn't generally group all enums together.

Reply:
`pyfluent_enums.py` ~-> `launch_options.py`

Reply:
Yes I agree on all points. @seanpearsonuk @prmukherj

Reply:
> `pyfluent_enums.py` ~-> `launch_options.py`

Will be done as part of: https://github.com/ansys/pyfluent/issues/4051